ios/FluentUI/PersonaButtonCarousel/MSFPersonaButtonCarousel.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+/// UIKit wrapper that exposes the SwiftUI PersonaButtonCarousel implementation
+@objc open class MSFPersonaButtonCarousel: NSObject, FluentUIWindowProvider {
+
+    // MARK: - Public API
+
+    @objc open var view: UIView {
+        return hostingController.view
+    }
+
+    @objc open var state: MSFPersonaCarouselState {
+        return self.personaButtonCarousel.state
+    }
+
+    @objc public init(size: MSFPersonaButtonSize = .large,
+                      theme: FluentUIStyle? = nil) {
```

Comment:
Same as the other comment about the convenience initializer to make objc calls less verbose.